pkg/job/runtime_v2/job/util/kuberuntime/kube_job.go

```diff
@@ -439,19 +439,28 @@ func generateContainerCommand(command string, workdir string) []string {
 	return commands
 }
 
-func GenerateResourceRequirements(flavour schema.Flavour) (corev1.ResourceRequirements, error) {
+func GenerateResourceRequirements(flavour, limitFlavour schema.Flavour) (corev1.ResourceRequirements, error) {
```

Comment:
It might be better to use `(request, limit schema.Flavour)` as parameters.

Reply:
done